src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -58,6 +59,7 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
```

Comment:
Unused

Reply:
?

Reply:
It picked the wrong line, one below LongStream